receiver/azureeventhubreceiver/azure_formatter_impl_test.go

```diff
@@ -0,0 +1,339 @@
+				ResultType:        &resultType,
+				ResultSignature:   &resultSignature,
+				ResultDescription: &resultDescription,
+				DurationMs:        &goodDuration,
+				CallerIPAddress:   &callerIPAddress,
+				CorrelationID:     &correlationID,
+				Identity:          &identity,
+				Level:             &level,
+				Location:          &location,
+				Properties:        &properties,
+			},
+			expected: map[string]interface{}{
+				azureTenantID:          "tenant.id",
+				azureOperationName:     "operation.name",
+				azureOperationVersion:  "operation.version",
+				azureCategory:          "category",
+				azureCorrelationID:     correlationID,
+				azureResultType:        "result.type",
+				azureResultSignature:   "result.signature",
+				azureResultDescription: "result.description",
+				azureDuration:          int64(1234),
+				netSockPeerAddr:        "127.0.0.1",
+				azureIdentity:          "someone",
+				cloudRegion:            "location",
+				cloudProvider:          "azure",
+				azureProperties:        properties,
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			assert.Equal(t, tt.expected, extractRawAttributes(tt.log))
+		})
+	}
+
+}
+
+var minimumLogRecord = func() plog.LogRecord {
+	lr := plog.NewLogs().ResourceLogs().AppendEmpty().ScopeLogs().AppendEmpty().LogRecords().AppendEmpty()
+
+	ts, _ := asTimestamp("2022-11-11T04:48:27.6767145Z")
+	lr.SetTimestamp(ts)
+	lr.Attributes().PutStr(azureOperationName, "SecretGet")
+	lr.Attributes().PutStr(azureCategory, "AuditEvent")
+	lr.Attributes().PutStr(cloudProvider, "azure")
+	lr.Attributes().Sort()
+	return lr
+}()
+
+var maximumLogRecord = func() plog.LogRecord {
+	lr := plog.NewLogs().ResourceLogs().AppendEmpty().ScopeLogs().AppendEmpty().LogRecords().AppendEmpty()
+
+	ts, _ := asTimestamp("2022-11-11T04:48:27.6767145Z")
+	lr.SetTimestamp(ts)
+	lr.SetSeverityNumber(plog.SeverityNumberWarn)
+	lr.SetSeverityText("Warning")
+	guid := "607964b6-41a5-4e24-a5db-db7aab3b9b34"
+
+	lr.Attributes().PutStr(azureTenantID, "/TENANT_ID")
+	lr.Attributes().PutStr(azureOperationName, "SecretGet")
+	lr.Attributes().PutStr(azureOperationVersion, "7.0")
+	lr.Attributes().PutStr(azureCategory, "AuditEvent")
+	lr.Attributes().PutStr(azureCorrelationID, guid)
+	lr.Attributes().PutStr(azureResultType, "Success")
+	lr.Attributes().PutStr(azureResultSignature, "Signature")
+	lr.Attributes().PutStr(azureResultDescription, "Description")
+	lr.Attributes().PutInt(azureDuration, 1234)
+	lr.Attributes().PutStr(netSockPeerAddr, "127.0.0.1")
+	lr.Attributes().PutStr(cloudRegion, "ukso")
+	lr.Attributes().PutStr(cloudProvider, "azure")
+
+	lr.Attributes().PutEmptyMap(azureIdentity).PutEmptyMap("claim").PutStr("oid", "607964b6-41a5-4e24-a5db-db7aab3b9b34")
+	m := lr.Attributes().PutEmptyMap(azureProperties)
+	m.PutStr("string", "string")
+	m.PutDouble("int", 429)
+	m.PutDouble("float", 3.14)
+	m.PutBool("bool", false)
+	m.Sort()
+
+	lr.Attributes().Sort()
+
+	return lr
+}()
```

Comment:
Could I ask you to move this to line 29 so it is defined above all the `Test*` functions, mainly so I don't need to jump back and forth.

Reply:
Done.